c-pallets/segment-book/src/lib.rs

```diff
@@ -248,7 +252,7 @@ pub mod pallet {
 		_,
 		Blake2_128Concat,
 		AccountOf<T>,
-		BoundedVec<ChallengeInfo<T>, T::StringLimit>,
+		BoundedVec<ChallengeInfo<T>, T::StringLimit>, // Again it seems StringLimit is used everywhere. Maybe it needs different limit?
```

Comment:
I configured a new limit for it:
https://github.com/CESSProject/cess/pull/88/files#diff-e315e38811b02c01aead9143bcfe8a7d0db178f24a154bad1431a08648393296R257